charts/access/discord/values.schema.json

```diff
@@ -0,0 +1,399 @@
+                }
+            ],
+            "required": [
+                "address",
+                "identitySecretName",
+                "identitySecretPath"
+            ],
+            "properties": {
+                "address": {
+                    "$id": "#/properties/teleport/properties/address",
+                    "type": "string",
+                    "default": "",
+                    "examples": [
+                        "auth.example.com:3025"
+                    ]
+                },
+                "identitySecretName": {
+                    "$id": "#/properties/teleport/properties/identitySecretName",
+                    "type": "string",
+                    "default": ""
+                },
+                "identitySecretPath": {
+                    "$id": "#/properties/teleport/properties/identitySecretPath",
+                    "type": "string",
+                    "default": "auth_id",
+                    "examples": [
+                        "auth_id"
+                    ]
+                }
+            },
+            "additionalProperties": true
+        },
+        "discord": {
+            "$id": "#/properties/discord",
+            "type": "object",
+            "default": {},
+            "examples": [
+                {
+                    "token": "example-api-token"
+                }
+            ],
+            "required": [
+                "token",
+                "tokenFromSecret",
+                "tokenSecretPath"
+            ],
+            "properties": {
+                "token": {
+                    "$id": "#/properties/discord/properties/token",
+                    "type": "string",
+                    "default": "",
+                    "examples": [
+                        "example-api-token"
+                    ]
+                },
+                "tokenFromSecret": {
+                    "$id": "#/properties/discord/properties/tokenFromSecret",
+                    "type": "string",
+                    "default": "",
+                    "examples": [
+                        "discord-token"
+                    ]
+                },
+                "tokenSecretPath": {
+                    "$id": "#/properties/discord/properties/tokenSecretPath",
+                    "type": "string",
+                    "default": "discordToken",
+                    "examples": [
+                        "token"
+                    ]
+                }
+            },
+            "additionalProperties": true
+        },
+        "roleToRecipients": {
+            "$id": "#/properties/roleToRecipients",
+            "type": "object",
+            "default": {},
+            "examples": [
+                {
+                    "dev": [
+                        "devs-discord-channel"
+                    ],
+                    "*": [
+                        "admin@email.com",
+                        "admin-discord-channel"
+                    ]
```

Comment:
:nail_care: 
I would try to keep the examples as close as possible to real values